lib/compiler/helpers/manual-restart.ts

```diff
@@ -0,0 +1,10 @@
+export function listenForManualRestart(callback: () => void) {
+  const stdinListener = (data: Buffer) => {
+    if (data.toString().replace('\n', '') === 'rs') {
```

Comment:
What about using `trim()` instead of replace? It might covers edge cases I guess

Reply:
Excellent idea! Done :heavy_check_mark: